lib/metasploit/framework/data_service/remote/http/error.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+module Metasploit
+  module Framework
+    module DataService
+      module Remote
+        class HttpError < StandardError
+          def initialize(error:, status_code:, message: 'Unknown Error')
+            super(message)
+            @error = error
+            # @message = message
```

Comment:
Why is this commented out? 👀

Reply:
I think we want to call `super(message)` or expose message as a method/attr_reader

Reply:
I added it in before realising I could just call `super(message)` commented it out and forgot to remove, not sure ifI was calling `super(message)` when you left the comment but it's certainly there now